Automattic/woocommerce-payments
# Make updating saved payment method async during checkout

Fixes #4440 

#### Changes proposed in this Pull Request
- Make updating saved payment method during checkout async for performance benefits.









#### Testing instructions





* Follow the normal checkout flow using a saved payment method.
* Ensure there is a scheduled action created as `wcpay_update_saved_payment_method` in Scheduled Actions (Tools > Scheduled Actions)
* Run the action and ensure there are no errors, additionally check logs to ensure a `POST wcpay/payment_methods/{{payment_method}}` call was made. 

-------------------


**Post merge**



- [x] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : [Testing instructions](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions-for-WC-Payments-4.5.0#make-updating-saved-payment-methods-async)
